bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableSession.java

```diff
@@ -707,4 +742,15 @@ public synchronized void close() throws IOException {
   public BigtableOptions getOptions() {
     return this.options;
   }
+
+  /**
+   * Getter for the field <code>gcjClientContext</code>.
+   *
+   * @return a HashMap of Connection Name (String) to {@link ClientContext} object.
+   */
+  public HashMap<String, ClientContext> getGCJClientContext() {
+    Preconditions.checkState(
+        options.useGCJClient(), "Client Context is only available for GCJ Client.");
+    return gcjClientContext;
+  }
```

Comment:
I m a little confused about these `clientContexts`, Could you please describe me where/how are we planning to use this?